packages/fx-core/src/plugins/solution/fx-solution/v2/getQuestions.ts

```diff
@@ -289,13 +292,27 @@ export async function getQuestions(
     } else {
       plugins = getAllV2ResourcePlugins();
     }
-    plugins = plugins.filter((plugin) => !!plugin.deploy && plugin.displayName !== "AAD");
-    if (plugins.length === 0 && inputs.skipAadDeploy !== "no") {
+
+    let filterAadPlugin = true;
+    if (
+      isAadManifestEnabled() &&
+      (inputs.platform === Platform.CLI_HELP || inputs.platform === Platform.CLI)
```

Comment:
What is this line for?

Reply:
We do not filter questions of AAD when cli or cli_help